duckduckgo.pot

```diff
@@ -6114,12 +6236,24 @@ msgctxt "settings"
 msgid "This will reset your saved settings to default values. Continue?"
 msgstr ""
 
+# smartling.placeholder_format=C
+#. Indicates this is the number of ties for this team
+msgctxt "Sports module"
+msgid "Ties"
+msgstr ""
+
 # smartling.placeholder_format=C
 #. The name of the Tigrinya language
 msgctxt "language_name"
 msgid "Tigrinya"
 msgstr ""
 
+# smartling.placeholder_format=C
+#. Part 1 of text that tells in which timezone the game times are displayed. Full text is: 'Times are in your local timezone'
+msgctxt "Sports module"
+msgid "Times are in your"
```

Comment:
@markapruett I'm afraid this will create problems for translations because of different sentence structure in other languages. I've checked out the markup but I'm not 100% clear on the possible options. Please ping me on MM so we can discuss how to solve this.